packages/devtools_app/test/memory/memory_screen_test.dart

```diff
@@ -794,5 +797,140 @@ void main() {
       },
       skip: true,
     );
+
+    group('Allocation Profile Table', skip: !enableNewAllocationProfileTable,
```

Comment:
instead of skipping, can we set `enableNewAllocationProfileTable` to true in setUp so that we can test the functionality in this PR? and so that we can ensure some other changes to the memory page do not break the work you've added here?